oak-run-elastic/pom.xml

```diff
@@ -221,7 +221,7 @@
         <dependency>
             <groupId>org.apache.tika</groupId>
             <artifactId>tika-core</artifactId>
-            <version>1.24.1</version>
+            <version>${tika.version}</version>
         </dependency>
```

Comment:
I believe it would be better to just remove the version tag here (I think than then decl in the parent pom applies)